src/hotspot/share/memory/filemap.cpp

```diff
@@ -1667,12 +1667,12 @@ char* FileMapInfo::map_bitmap_region() {
   char* bitmap_base = os::map_memory(_fd, _full_path, si->file_offset(),
                                      requested_addr, si->used_aligned(), read_only, allow_exec, mtClassShared);
   if (bitmap_base == NULL) {
-    log_error(cds)("failed to map relocation bitmap");
+    log_info(cds)("failed to map relocation bitmap");
     return NULL;
   }
 
   if (VerifySharedSpaces && !region_crc_check(bitmap_base, si->used_aligned(), si->crc())) {
-    log_error(cds)("relocation bitmap CRC error");
+    log_info(cds)("relocation bitmap CRC error");
```

Comment:
I think this one should remain an error -- it means the CDS archive has been corrupted.

Reply:
I reverted it.